src/apps/config/support.lua

```diff
@@ -98,11 +98,20 @@ local function compute_objects_maybe_updated_in_place (schema_name, config,
    return objs
 end
 
-local function record_mutable_objects_embedded_in_app_initarg (name, obj, accum)
+local function record_mutable_objects_embedded_in_app_initarg (id, name, obj, accum)
    local function record(obj)
       local tab = accum[obj]
-      if not tab then tab = {}; accum[obj] = tab end
-      table.insert(tab, name)
+      local debugit = true
```

Comment:
What's this?

Reply:
Left over debug statement, clearly got through my pre-push check. I'll remove it :)